cmd/recording.go

```diff
@@ -0,0 +1,126 @@
+package cmd
```

Comment:
The name of this file should match the subcommand being implemented (`appmap.go`, rather than `recording.go`).

Reply:
Actually, I would like to see the command be called `recording` rather than `appmap`. So `appmap recording start,stop,check`